tasks/concurrent.js

```diff
@@ -4,16 +4,20 @@ var padStream = require('pad-stream');
 var async = require('async');
 var arrify = require('arrify');
 var indentString = require('indent-string');
+var chalk = require('chalk');
 
 var cpCache = [];
 
+var colors = ["blue","magenta","yellow","green","red","cyan","white","gray","redBright","greenBright","yellowBright","blueBright","magentaBright","cyanBright"];
```

Comment:
the way I did it now is that it uses all those colors and if all are used it uses no color. 
I thought of mapping the name to a color with color-hash, but the colors might not be distinguishable / usable. If we leave out the bright ones that would give 8 colors, and from 9 on it would be the default color. Acceptable? Or do you have a better idea?